package.json

```diff
@@ -2,14 +2,37 @@
   "name": "@govtechsg/tradetrust-utils",
   "version": "0.1.0",
   "description": "",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/types/src/index.d.ts",
+  "types": "./dist/types/index.d.ts",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/types/index.d.ts"
+      ],
+      "constants/*": [
+        "./dist/types/constants/*.d.ts"
+      ]
+    }
+  },
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "import": "./dist/esm/index.js"
+    },
+    "./constants/*": {
+      "require": "./dist/cjs/constants/*.js",
+      "import": "./dist/esm/constants/*.js"
+    }
+  },
```

Comment:
Nice! Have you tried importing into another project with these settings? Are these paths recognised in the other project when you import?

Reply:
yeah they do, autocompletes too